environment.yml

```diff
@@ -9,7 +9,6 @@ dependencies:
 - jsonschema=2.5.1
 - jupyter=1.0.0
 - libffi=3.2.1
-- libgcc=5.2.0
```

Comment:
FWIW, `libgcc` 4.8.5 is available on OS X from defaults, not that this matters. If we do not need `libgcc` as a runtime dependence, then we should get rid of it.